apps/charterafrica/src/lib/data/common/index.js

```diff
@@ -94,14 +94,25 @@ async function getFeaturedConsultations(consultation, playlistItems) {
     }
     return [];
   }
-  const featured = consultation.featured?.map((item) =>
-    playlistItems.find((plItem) => plItem.videoId === item.videoId)
-  );
+  const featured =
+    consultation?.featured?.map((item) =>
+      playlistItems.find((plItem) => plItem.videoId === item.videoId)
+    ) || [];
   return featured;
 }
 
 async function processPageConsultation(page) {
   const { blocks } = page;
+
+  const { documents } =
+    blocks.find(({ slug }) => slug === "consultation-documents") || null;
+  if (documents) {
+    blocks.push({
+      slug: "documents",
+      ...documents,
+    });
```

Comment:
Don't add a new block; just update the existing consultation-documents (similar to how it's done in consultation-multimedia).